docs/persistence/README.md

```diff
@@ -0,0 +1,160 @@
+# Persistent Data
+
+In this section we will discuss how to work with persistent data in your
+applications and the different options available to you.
+
+Persistent data is data that is stored on disk and survives application
+restarts. This is in contrast to ephemeral data which is stored in memory
+and is lost when the application is restarted.
+
+## Responsibilities
+
+The team is reponsible for any data that is stored in the various storage
+options that are available through the platform. You can read more in the
+[Data Responsibilities](./responsibilities.md) section.
+
+## What should I choose?
+
+Sequence of questions to ask yourself when choosing the right storage option.
+Choose wisely.
+
+```mermaid
+graph TD
+  A[I got data!] --> B[Is it structured?]
+  B --> |Yes| C[Is it events?]
+  B --> |No| D[Is it files?]
+  C --> |Yes| Kafka
+  C --> |No| E[Is it analytical?]
+  D --> |Yes| GCS[Cloud Storage]
+  D --> |No| Opensearch[OpenSearch]
+  E --> |Yes| GBQ[BigQuery]
+  E --> |No| GCSQL[Cloud SQL]
+
+  click GBQ "#bigquery"
+  click GCS "#cloud-storage"
+  click GCSQL "#cloud-sql"
+  click Kafka "#kafka"
+  click Opensearch "#opensearch"
+```
+
+## Storage Comparison
+
+Below is a list of the different storage options available to you.
+
+| Name | Type | Availebility |  Environment | Backup |
+| ---- | ---- | ------------ |  ----------- | ------ |
+| [Kafka](#kafka) | Streaming | ✅ | All | Yes* |
+| [Cloud Storage](#cloud-storage) | Object | ✅ | GCP | Yes* |
+| [Cloud SQL](#cloud-sql) | Relational | ✅ | GCP | Yes |
+| [BigQuery](#bigquery) | Relational | ✅ | GCP | Yes* |
+| [Elasticsearch](#elasticsearch) | Document | ⚠️ | All | Yes |
+| [OpenSearch](#opensearch) | Document | ✅ | All | Yes |
+| [InfluxDB](#influxdb) | Time Series | ⚠️ | All | Yes |
+| [Redis](#redis) | Key Value | ⚠️ | All | No |
+| [IBM MQ](#ibm-mq) | Message | ⚠️ | All | Yes* |
+| [MongoDB](#mongodb) | Document | ⚠️ | GCP | No |
+
+\* Data is highly aviailable and fault tolerent but not backed up if deleted by
+mistake.
+
+## Kafka
+
+Kafka is a streaming platform that is used for storing and processing data. It
+is a very powerful tool that can be used for a wide variety of use cases. It is
+also a very complex tool that requires a lot of knowledge to use effectively.
+
+[:octicons-arrow-right-24: Getting started with Kafka](./kafka/README.md)
+
+## Cloud Storage
```

Comment:
Legge på "(Buckets)" for å gjøre det lettere for de som ikke kan det offisielle navnet?